DirectEmployers/MyJobs
# [PD 2855] metric sums

Task was to add the metric sums at the bottom of the table

To Test:

Pull down the repository. You can see at the bottom that the Showing is displayed for how many are showed on a specific page along with the grand total. You can change pages and also change the amount per page and it will update accordingly.
